client/angular/controllers/MainController.js

```diff
@@ -502,14 +478,6 @@
                     "width": tweetWidth + "px",
                 };
             }
-            return {
```

Comment:
Maybe make this return an empty string in this case? Would probably make a bit more sense, even though it seems to work fine like this.